### DISCUSSION TITLE
Question - How long does the webhook service run

### DISCUSSION THREAD

**Discussion by Pilaarbijter** [adnanh/webhook#679] [General]
Hello,

I have a pretty easy question to ask, how long is the webhook service 'active' and able to receive requests when ran on a Pi? 

Thanks for the reply!

**Comment by moorereason** [comment]
It can run forever, but it depends on how you run the service.  The fact that it's running on a Pi doesn't matter.  What matters is the OS and service management framework you use.

**Comment by Pilaarbijter** [comment]
Well the reason I ask is that after a certain amount of time, the http webhook doesn't work anymore. Almost as if the service is no longer running.

I'm running it on a Debian OS. As of your question tot the service management framework, I'm not sure I'm following. Can I query this information via the command line?

**Reply by moorereason** [reply]
There's no reason for webhook to stop responding, so I assuming it's something on your system.

I'm assuming you're using systemd to manage the service on Debian.  I don't have time to explain how systemd works, but look into the status of that service (see `systemctl`).  Check your system logs.  Do SysAdmin things. :grin:
